src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/OnAutoInsertTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.VisualStudio.Razor.IntegrationTests
+{
+    public class OnAutoInsertTests : AbstractRazorEditorTest
+    {
+        [IdeFact]
+        public async Task CSharp_DocumentationComments()
```

Comment:
Literally no idea if this passes.. can't run the tests on my machine, they don't run on CI, and they fail the nightly run. What a world!

Reply:
I forgot, what was the reason we aren't running integration tests on CI right now? Is it just due to failures/flakiness?

Reply:
Originally it was because we were ahead of the VS version being used, I thought, but no idea if that is still the blocker, because there are certainly infra issues.